src/connectors/tenable.py

```diff
@@ -0,0 +1,118 @@
+"""Tenable
+Collects a variety of Tenable logs into tables
+"""
+from tenable.io import TenableIO
+from datetime import datetime
+
+from runners.helpers import db
+from runners.helpers.dbconfig import ROLE as SA_ROLE
+
+CONNECTION_OPTIONS = [
```

Comment:
So right now this will only have one option (and it should be the default), but might it make sense to make this ahead-of-time extensible with a `connection_type` with value `user`?